iceoryx_posh/test/moduletests/test_posh_runtime.cpp

```diff
@@ -517,6 +517,22 @@ TEST_F(PoshRuntime_test, GetMiddlewareSubscriberWithQueueCapacityZeroClampsQueue
     EXPECT_EQ(1U, subscriberPort->m_chunkReceiverData.m_queue.capacity());
 }
 
+TEST_F(PoshRuntime_test, GetMiddlewareSubscriberWithHistoryRequestLargerThanQueueCapacityClampsToQueueCapacity)
+{
+    ::testing::Test::RecordProperty("TEST_ID", "77ca8d29-ffcb-4860-bf07-0af30b352e5c");
+    iox::popo::SubscriberOptions subscriberOptions;
+    constexpr uint64_t EXPECTED_HISTORY_REQUEST = 1U;
```

Comment:
Maybe we should test this with a non-default queue capacity > 1?

Reply:
The default `queueCapacity` is not 1 but `SubscriberChunkQueueData_t::MAX_CAPACITY`.

Reply:
Yes, true. I still do not quite like the one and would prefer some other number.